### PR TITLE
ADEN-6047 JWPlayer with A9 bidder

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js [hunk 1]
```diff
@@ -50,15 +50,24 @@ define('wikia.articleVideo.featuredVideo.ads', [
...
 
+		log(['buildVastUrl', position, videoDepth, slotParams, options], log.levels.debug, logGroup);
+
 		return vastUrlBuilder.build(aspectRatio, slotParams, options);
 	}
 
-	return function (player) {
+	return function (player, bidParams) {
 		var correlator,
+			vastUrl,
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js]: Do we need this variable to be in parent scope?

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js [hunk 2]
```diff
@@ -50,15 +50,24 @@ define('wikia.articleVideo.featuredVideo.ads', [
 				src: 'premium'
 			};
 
+		if (videoDepth === 1) {
+			Object.keys(bidParams).forEach(function (key) {
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js]: please make sure `bidParams` isn't `undefined`

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 3]
```diff
@@ -31,27 +37,41 @@ require([
...
-	});
+		featuredVideoAds(playerInstance, bidParams);
+		featuredVideoTracking(playerInstance, willAutoplay);
+		handleTabNotActive(willAutoplay);
+	}
 
-	featuredVideoAds(playerInstance);
-	featuredVideoTracking(playerInstance, willAutoplay);
-	handleTabNotActive(willAutoplay);
+	if (a9 && adContext.get('bidders.a9Video')) {
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js:63]: Is it possible to move logic related to `a9` into `featuredVideoAds` module?

Comment (fraszczakszymon) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js:63]: Unfortunately we can't add ads related event listener before instance is created

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**